data/faq/en/home.yaml

```diff
@@ -1,3 +1,19 @@
+- question: Is SimpleLogin disposable mail provider?
+  answer: >
+  No! SimpleLogin is <b>not</b> disposable mail because
+  
+  - You need an account to use SimpleLogin
+ 
+  - All aliases generated are unique
+  
+  - Multiple anti-abuse automation
+ 
+  - Anti-SPAM limitations
```

Comment:
I think we can remove this point as it's kind of similar to the previous one and not easily understandable